File: Lab 1 Solutions/Program9.c
#include <stdio.h>

void swap(int *m, int *n){
	int temp = *m;
	*m = *n;
	*n = temp;
}

int main(){
	int a, b;
	printf("Enter any two integers: ");
	scanf("%d %d", &a, &b);
	swap(&a, &b);
	printf("The numbers after swapping are : %d & %d.", a, b);
	return 0;
}

File: Lab 1 Solutions/Program9.py
num1, num2 = input("Enter any two numbers: ").split()

def swap(a, b):
    a ,b = b, a
    return (a, b)

num1, num2 = swap(num1, num2)
print("The numbers after swapping are: "+ num1 +" & " + num2)
